neon-animated-pages.sublime-snippet

```diff
@@ -1,9 +1,9 @@
 <snippet>
     <content><![CDATA[
-<neon-animated-pages selected="${1:0}" entry-animation="${1:slide-from-fright-animation}" exit-animation="${2:slide-left-animation}">
-    <neon-animatable>$3</neon-animatable>
+<neon-animated-pages selected="${1:0}" entry-animation="${2:slide-from-fright-animation}" exit-animation="${3:slide-left-animation}">
```

Comment:
Yes,this too
Am 19.04.2016 16:32 schrieb "Rob Dodson" <notifications@github.com>:

> In neon-animated-pages.sublime-snippet
> <https://github.com/robdodson/PolymerSnippets/pull/45#discussion_r60241789>
> :
>
> > @@ -1,9 +1,9 @@
> >  <snippet>
> >      <content><![CDATA[
> > -<neon-animated-pages selected="${1:0}" entry-animation="${1:slide-from-fright-animation}" exit-animation="${2:slide-left-animation}">
> > -    <neon-animatable>$3</neon-animatable>
> > +<neon-animated-pages selected="${1:0}" entry-animation="${2:slide-from-fright-animation}" exit-animation="${3:slide-left-animation}">
>
> slide-from-right maybe?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/robdodson/PolymerSnippets/pull/45/files/e2850cedcce898f87565d24ccb3ce9f5f115c466#r60241789>
>